python/src/aac/plugins/first_party/primitive_type_check/primitive_type_check.yaml

```diff
@@ -0,0 +1,5 @@
+model:
+  name: primitive-type-checking
+  description: |
+    Primitive Type Checking is an Architecture-as-Code plugin that provides
+    primitive type validations for the primitive enum types defined in the core specification.
```

Comment:
Is this complete? It seems like it's missing `behaviors`.

Reply:
Becuase this plugin doesn't have any behaviors (commands). I can add a behavior for the validations that describes what they do, but I don't think it'll transfer well to a plugin-specific model if we have specific fields for commands rather than behaviors. I'll spend some time thinking on that.